guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<blockquote><a href="https://medium.com/mysuperai/what-is-named-entity-recognition-ner-and-how-can-i-use-it-2b68cf6f545d" target="_blank">Named Entity Recognition</a>&nbsp;is a branch of information extraction. This is used to identify entities such as&nbsp;<strong>"Organizations"&nbsp;</strong>,&nbsp;<strong>"Person"&nbsp;</strong>,&nbsp;<strong>"Date"&nbsp;</strong>,&nbsp;<strong>"Country"</strong>, etc. that are present in the text.</blockquote><blockquote><br></blockquote><p>Need to remove extra spaces before comma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
seems like a rendering issue, spaces before comma are not present in the above mentioned text